### ISSUE TITLE
example is offline?

### ISSUE THREAD

**Issue by hems** [olragon/meteor-apidoc#1]
http://apidoc.meteor.com/document/D8cp85WGYxetcspd4/preview

**Comment by jamiter** [comment]
All Meteor free hosting sites are offline. They stopped supporting the free hosting: https://forums.meteor.com/t/meteor-com-free-hosting-ends-march-25-2016/19308

**Comment by hems** [comment]
cool, shouldn't we remove it from the readme then or host it at some free box at heroku or something?

**Comment by jamiter** [comment]
Yup, maybe the maintainer @olragon could do that. But I'm not sure if this is still being maintained. Doesn't look like it. Last commit was a year ago.
